queens-psyc-clinic/app
# FE Add labels to navbar

### Edited Navbar Component
Following up from Professor Powley's comments, we should add titles to the navbar as the icons are a bit unclear. So this component was adjusted to include titles and I also made width a little wider for a nicer layout.

### Testing
1. navigate to branch: `git checkout navbar-title`
2. cd to the public folder
3. run `npm install`
4. run `npm start`
5. ensure the navbar is showing with the titles and a little wider (now 6rem) and no errors!

<img width="100" alt="Screenshot 2024-03-01 at 12 33 52 PM" src="https://github.com/queens-psyc-clinic/app/assets/74020381/c03f29d4-0d03-4236-9f0e-6d00c91900f6">


For reference checkout #68
